### PR TITLE
Renamed every instance of "plastique" (c4 explosives) to "c4"

### PR DESCRIPTION
**Pull request by Irradiation**
This is in the name of every admin out here and anybody doing testing. Fuck you old c*ders.

[tweak][qol]

:cl:
 * tweak: C4 is now properly named as such in the code (previously called "plastique").

### CODE REVIEW

File: maps/NRVHorizon.dmm [hunk 1]
```diff
@@ -4723,7 +4685,7 @@
 "bMQ" = (/obj/machinery/door_control{id_tag = "Dorm2"; name = "Dorm 2 Door Bolt Control"; normaldoorcontrol = 1; pixel_x = 0; pixel_y = 24; req_access_txt = "0"; specialfunctions = 4},/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden/layered{dir = 4},/obj/machinery/atmospherics/pipe/simple/supply/hidden/layered{dir = 10},/turf/simulated/floor/wood,/area/crew_quarters/sleep)
 "bMR" = (/obj/structure/table,/obj/machinery/atmospherics/unary/vent_scrubber/on/layered{dir = 8},/obj/machinery/light/small{dir = 1},/turf/simulated/floor/wood,/area/crew_quarters/sleep)
 "bMS" = (/obj/effect/landmark{name = "xeno_spawn"; pixel_x = -1},/turf/simulated/floor/engine,/area/science/hallway)
-"bMT" = (/obj/structure/table/reinforced,/obj/item/weapon/plastique{pixel_x = 3},/obj/item/weapon/plastique,/turf/simulated/shuttle/floor/vox{icon_state = "floor4"},/area/shuttle/vox/station)
```
Comment (Kurfursten) [inline maps/NRVHorizon.dmm:4726]: It looks like you just deleted these instead of replacing them with c4, sneaky mapper

File: maps/test_box.dmm [hunk 2]
```diff
@@ -8013,7 +8013,7 @@
 "cYe" = (/obj/machinery/door/poddoor{id_tag = "QMLoaddoor2"; name = "Supply Shuttle Loading Door"},/obj/machinery/conveyor{dir = 4; id_tag = "QMLoad2"},/turf/simulated/shuttle/plating,/area/shuttle/supply)
 "cYf" = (/obj/machinery/conveyor{dir = 4; id_tag = "anosample"},/turf/unsimulated/floor{name = "plating"},/area/centcom/suppy)
 "cYg" = (/obj/machinery/conveyor{dir = 4; id_tag = "anosample"},/obj/structure/plasticflaps/mining,/turf/unsimulated/floor{name = "plating"},/area/centcom/suppy)
-"cYh" = (/obj/structure/closet/bombcloset,/obj/item/weapon/plastique,/obj/item/weapon/plastique,/obj/item/weapon/plastique,/obj/item/weapon/plastique,/obj/item/weapon/plastique,/obj/item/weapon/plastique,/obj/effect/spawner/newbomb/timer,/obj/item/clothing/head/advancedeod_helmet,/obj/item/clothing/suit/advancedeod,/turf/simulated/shuttle/floor,/area/shuttle/arrival/station)
```
Comment (Kurfursten) [inline maps/test_box.dmm:8016]: same but it's on a test map so no big deal

**Comment by ancientpower** [comment]
please install an ide and mass change it so this doesn't pop up later on

File: maps/NRVHorizon.dmm (hunk 1 above)
Comment (Irradiation) [reply maps/NRVHorizon.dmm]: The map editor to fix this issue is garbo to say the least, but I'll see around to putting it back in...wherever the fuck it was.

File: maps/test_box.dmm (hunk 2 above)
Comment (Irradiation) [reply maps/test_box.dmm]: Righto, I'll unfuck that tomorrow

**Comment by small-boss2** [comment]
thank god holy shit

**Pull request merged**